Doc/library/math.rst

```diff
@@ -199,8 +199,8 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return ``e**x``.
-
+   Return e raised to the power *x*, where e = 2.718281... is the base
```

Comment:
I think *e* should be emphasized, like in `log()`, `log1p`, etc.  Maybe `e = 2.718281...` should have the code markup.